### ISSUE TITLE
Options for different fuel-consumption units

### ISSUE THREAD

**Issue by gthbusrr** [akhilrex/hammond#40]
Currently, fuel-consumption are only available at `km/L` (I don't see a way to change that at least). It would be great if another units like `L/100km`, `mpg`, etc are available.

Thanks ;)

**Comment by akhilrex** [comment]
The fuel consumption is always going to be your chosen distance unit and chosen fuel quantity unit. If you had setup your default distance and volume units as miles and gallons you would have seen this in mpgs.

I will consider building conversions to other units in future.

**Comment by Atwistoflemming** [comment]
In the UK petrol and diesel is sold in Litres, but vehicle fuel consumption is often measured (or compared) using miles per gallon (mpg). Clarkson seemed to handle dual fuel quantity units, so would appreciate if this feature could be added, thanks.

PS Recognise many may prefer fuel consumption in Litres per 100km (and this could be set as a default), but I prefer to have the option of imperial.

**Comment by StefanescuCristian** [comment]
I agree with @Atwistoflemming. I think that in all Europe L/100km is somewhat standard.

**Comment by djjudas21** [comment]
+1 for dual unit support. As @Atwistoflemming says, fuel in the UK is sold in litres but fuel consumption is is stated in miles per gallon.

**Comment by geekblog-dev** [comment]
+1 For Unit selection
It would be so much more handy if can see Stats for my Cars in l/100km.

**Comment by SEEN88** [comment]
Another +1 for different fuel consumption metrics, here in New Zealand we also use L/100km, this is the only thing I miss from Clarkson so far.

**Comment by AlfHou** [comment]
I'll take a look at adding this.

**Comment by chust4** [comment]
+1 for L/100km
